src/scss/components/search/_search.scss

```
@@ -0,0 +1,33 @@
+@charset "UTF-8";
+
+@include grid-media-min('md') {
+  .su-site-search {
+    display: none;
+  }
+
+  .su-multi-menu {
+    display: flex;
+  
+    .menu-search-link {
+      margin-left: 3.6rem;
+      align-self: center;
+
+      i {
+        color: $su-color-cardinal-red-dark;
+        transform: scaleX(-1);
+
+        &:hover,
+        &:focus,
+        &:active {
+          color: $su-color-black;
+        }
+      }
+    }
+  }
+}
+
+@include grid-media-max('md') {
+  .menu-search-button {
```

```suggestion
  .menu-search-link {
```
Just gotta update this to the correct target class and we should be good to go! 😄